CHANGELOG.md

```diff
@@ -76,6 +76,7 @@
 - Added support for `Index.is_boolean`, `Index.is_integer`, `Index.is_floating`, `Index.is_numeric`, and `Index.is_object`.
 - Added support for `DatetimeIndex.round`, `DatetimeIndex.floor` and `DatetimeIndex.ceil`.
 - Added support for `Series.dt.days_in_month` and `Series.dt.daysinmonth`.
+- Added support for `DataFrameGroupBy.value_counts` and `SeriesGroupBy.value_counts`.
```

Comment:
I found we have a `value_counts` method in general.py, do you support that too?

Reply:
I'm not sure why we don't support the general.py method actually, since that uses the existing `Series.value_counts` implementation. Looking at the comments in our code + the pandas website, it seems like `pd.value_counts` is actually undocumented: https://pandas.pydata.org/docs/reference/general_functions.html

It should be simple to implement, but I'll leave that for a separate PR.

Reply:
SG. Let's do it then.